src/lib/components/FilterForm.tsx

```diff
@@ -66,19 +73,21 @@ export const FilterForm: React.FC<IFilterFormProps> = (props) => {
             <Grid container>
               <Grid item xs={2} style={{ marginRight: '-30px' }} />
               <Grid item xs={10}>
-                <FormControlLabel
-                  style={{ width: '100%' }}
-                  control={
-                    <Checkbox
-                      id="All"
-                      checked={allChecked}
-                      onChange={onCheckboxAllChange}
-                      name="All"
-                    />
-                  }
-                  label="Всі"
-                  key="All"
-                />
+                {allChecked !== undefined && (
```

Comment:
you cna use just 
```suggestion
                {!!allChecked && (
```

Reply:
here i need to distinguish between `false` and `undefined`  – when it's toggled to false vs when this prop is not passed by parent component